functional/lcow_test.go

```diff
@@ -22,7 +23,91 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
-func TestV2XenonLCOW(t *testing.T) {
+// TestLCOWUVMNoSCSISingleVPMem starts an LCOW utility VM without a SCSI controller and
+// only a single VPMem device. Uses initRD
+func TestLCOWUVMNoSCSISingleVPMem(t *testing.T) {
+	opts := &uvm.UVMOptions{
+		OperatingSystem:  "linux",
+		ID:               "uvm",
+		VPMemDeviceCount: 1,
+		NoSCSI:           true,
```

Comment:
Should we expose it this way? When we support multiple controllers a single boolean wont suffice

Reply:
Sure, will update to *int where nil means 1 by default.

Reply:
Yes I like that